src/auro-datepicker.js

```diff
@@ -361,6 +396,9 @@ export class AuroDatePicker extends LitElement {
 
     this.dropdown.addEventListener('auroDropdown-toggled', () => {
       this.setAttribute('aria-expanded', this.dropdown.isPopoverVisible);
+      if (this.dropdown.isPopoverVisible) {
+        this.notifyDatepickerToggled();
+      }
```

Comment:
We should fire this event when the bib is opened and when it's closed. The event includes the detail about `expanded`.

I would just remove the if statement wrapping this.